src/libraries/Microsoft.NET.WebAssembly.Webcil/src/Webcil/WebcilConverter.cs

```diff
@@ -223,12 +223,12 @@ private static void CopySections(Stream outStream, Stream inputStream, Immutable
     }
 
 #if NETCOREAPP2_1_OR_GREATER
```

Comment:
Not related to your PR, but this is a strange choice of a compilation constant... Stream.ReadExactly was added in .NET 7, not .NET Core 2.1.